packages/terra-form-select/src/SingleSelect.jsx

```diff
@@ -170,41 +166,40 @@ class SingleSelect extends React.Component {
 
   render() {
     const {
-      allowClear,
       children,
       defaultValue,
       onChange,
-      placeholder,
       required,
       value,
       intl,
+      isFilterStyle,
       ...otherProps
     } = this.props;
 
-    const defaultPlaceholder = intl.formatMessage({ id: 'Terra.form.select.defaultDisplay' });
-    const selectPlaceholder = placeholder === undefined ? defaultPlaceholder : placeholder;
-    let clearOptionDisplay;
+    let frameValue;
```

Comment:
nit: This can be renamed something different.

Reply:
Updated here https://github.com/cerner/terra-core/pull/3094/commits/7f7548bb813d8756aaa233b31d5ae4965d81d33d.